src/navigation.ts

```diff
@@ -1343,9 +1340,9 @@ export class Navigation {
     block: Blockly.BlockSvg,
   ): boolean {
     let isHandled = false;
-    const markedNode = workspace.getMarker(this.MARKER_NAME)?.getCurNode();
+    const markedNode = workspace.getCursor()?.getCurNode();
```

Comment:
Maybe `cursorNode` or `focusedNode` instead of `markedNode`?

Reply:
Renamed to `targetNode`.